actions/archive2disk/v1/README.md

```diff
@@ -20,15 +20,44 @@ Archive types supported:
 -   `tar`: standard tar archive
 -   `targz`: A gzipped tar archive (compressed)
 
+Optional SHASUM verification on the TAR file (not the tar.gz file), see usage examples below and more details at
```

Comment:
This seems like a weird place to do the shasum verification. Why would we want to decompress a potentially corrupted file and then check if its valid?

Reply:
For optimal speed, I just added on the the streaming / extracting process to additionally perform the checksum.  It would be slower to stream the file for a checksum verification then stream the file for extraction.  Another option may be to download the file then perform the checksum then extract and delete the file at the end upon successful checksum/extraction.  Though, where would you place the downloaded file?  The same location as the destination path?